rollup/internal/controller/relayer/l2_relayer.go

```diff
@@ -1118,6 +1127,7 @@ func (r *Layer2Relayer) skipSubmitByFee(oldest time.Time, metrics *l2RelayerMetr
 		)
 	}
 
+	r.metrics.rollupL2RelayerCommitPrice.Set(target)
```

Comment:
Should we track this metric when `forceSubmit == true`?

Reply:
true, thanks.